src/main/kubernetes/artemis/configmaps/artemis-configmap.yml

```diff
@@ -1,12 +1,14 @@
 apiVersion: v1
 data:
+  SPRING_ARTEMIS_HOST: activemq-artemis
   SPRING_DATASOURCE_URL: jdbc:mysql://artemis-mysql/artemis?useUnicode=true&characterEncoding=utf8&useSSL=false&useLegacyDatetimeCode=false&serverTimezone=UTC&createDatabaseIfNotExist=true
-  SPRING_PROFILES_ACTIVE: prod,bamboo,bitbucket,jira,artemis,scheduling
+  SPRING_PROFILES_ACTIVE: prod,artemis,scheduling,bamboo,bitbucket
```

Comment:
What is the reason behind removing jira? If you have the rest of the Atlassian suite, I'd expect Jira being active too.

Reply:
Maybe you can elaborate on this how you got to the active profiles in general as they are not the same throughout the config files

Reply:
You are right, jira needs to be there. I have accidentally removed it while testing different configs. I will revert the change.

Regarding the profiles - jira, bamboo, bitbucket is added to the artemis server since this is the application that connects to them, other microservices do not connect to them => profiles are not needed. The artemis profile is added to the Artemis server and the microservices since they relly on configs from application-artemis file. Other applications - JHipster Registry, ActiveMQ Artemis do not use artemis specific data, neither connect to Jira, Bamboo, Bitbicket => they are not added